main.cpp

```diff
@@ -133,256 +143,329 @@ struct CarWash
+{
+    // should I write it like this or how it was above? I feel like this way is less confusing with the true value being first spot in the ternary opeator after the question mark.
+    return needmaintenance == false ? true : false;
+}
+
 struct Brakes
 {
-    // type
     char type = 'B';
-    // brand
     std::string brand = "Acme";
-    // material
     std::string material = "Rubber";
-    // amount of pad left
     float padRemaining = 0.2f;
-    // need maintenance?
     bool needmaintenance = true;
 
-    // 3 things it can do:
-    // 1) stop the bike
-    void stop();
-    // 2) slow the bike
-    void slow();
-    // 3) squeak when needing to be replaced
-    bool replaceSqueek(int padRemaining); // returns a boolean telling if the pads need replaced 
+    bool stop(bool needMaintence, int pressureApplied);
+    float slow(float pressureApplied);
+    bool replaceSqueek(float padRemaining);
 };
 
+bool stop(bool needMaintence, float pressureApplied)
+{
+    return (needMaintence == false && pressureApplied > 0) ? true : false; 
+}
+
+float slow(float pressureApplied)
+{
+    return pressureApplied * 0.2f;
+}
+
+bool replaceSqueek(float padRemaining)
+{
+    return (padRemaining < 0.5f) ? true : false;
+}
+
 struct Handlebars
 {
-    // type
     char type = 'D';
-    // material
     std::string material = "Carbon Fiber";
-    // width
     float width = 7.9f;
-    // brand
     std::string brand = "Acme";
-    // need maintenance?
-    bool needmaintenance = false;
+    bool needMaintenance = false;
 
-    // 3 things it can do:
-    // 1) turn the bike
-    bool turn(); // returns if the bike will turn or not
-    // 2) engage the brakes
-    bool engageBrakes();  // returns if the bike will brake or not
-    // 3) shift the gears
+    float turn(bool needMaintenance, float turnDegree); 
+    bool engageBrakes(bool needMaintence, float pressureApplied); 
     void shiftGears(int desiredGear);
 };
 
+float turn(bool needMaintenance, float turnDegree)
+{
+    return needMaintenance ? turnDegree : 0.0f;
+}
+
+bool engageBrakes(bool needMaintence, float pressureApplied)
+{
+    return (needMaintence == false && pressureApplied > 0.0f) ? true : false;
+}
+
 struct Bicycle
 {
-    // Gears
     Gears gears;
-    // Frame
     Frame frame;
-    // Wheel
     Wheel wheel;
-    // Brakes
     Brakes brakes;
-    // Handlebars
     Handlebars handlebars;
 
-    // 3 things it can do:
-    // 1) shift gears
-    void shiftGears(int desiredGear);
-    // 2) pedal forward
-    void pedalForward();
-    // 3) brake
-    void brake();
+    bool shiftGears(int desiredGear);
+    bool pedalForward(int pedalSpeed);
+    bool brake(int appliedPressure);
 };
 
+bool shiftGears(int desiredGear, int currentGear)
+{
+    return (desiredGear != currentGear) ? true : false;
+}
 
+bool pedalForward(int pedalSpeed)
+{
+    return pedalSpeed > 0 ? true : false;
+}
 
-
-
+bool brake(int appliedPressure)
+{
+    return appliedPressure > 0 ? true : false;
+}
```

Comment:
These are not member functions.  these are free functions.
You did not fully qualify the function name
- [ ] revise